src/path-transformers.ts

```diff
@@ -53,12 +53,22 @@ export function urlFromPath(root: string, filepath: string) {
   const isPlatformWin = /^win/.test(process.platform);
   const isExtendedLengthPath = /^\\\\\?\\/.test(filepath);
   const hasNonAscii = /[^\x00-\x80]+/.test(filepath);
+  const stack = (() => {
+    try {
+      throw new Error('meh');
+    } catch (e) {
+      return e.stack;
+    }
+  })();
   if (isPlatformWin && !isExtendedLengthPath && !hasNonAscii) {
-    return path.win32.relative(root, filepath).replace(/\\/g, '/');
+    const relative = path.win32.relative(root, filepath).replace(/\\/g, '/');
+    console.log(`HEY WHATSUP ${root} is root and ${filepath
```

Comment:
Yeah those console.log in my commits were my LAAAAZY way to debug on windows since I knew I'd see the results in Appvveyor. :-p  Now that I have my Windows machine I won't do that again.